website/docs/components/code-editor/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,107 @@
+
+![Code Editor with rounded corners for enabled standalone property](/assets/components/code-editor/code-editor-rounded-standalone.png)
+
+Sometimes it may be necessary to use the Code Editor in a more dense layout or nested within another component.
+
+![Code Editor embedded inside of a logs UI, where the corners are squared to fit within container.](/assets/components/code-editor/code-editor-block-level.png)
+
+## Header
+
+An optional header is available within the Code Editor. It consists of three sections that can be toggled as needed:
+
+- the text content that includes the title and description
+- an actions container for secondary actions
+- a custom yielded section of primary actions.
+
+### When to use a title and description
+
+A title and description provides additional contextual information for the Code Editor. An accessible name is mandatory and can either be provided by the title or by an external text element.
+
+!!! Do
+
+Provide an accessible name for the Code Editor so that users of assistive technology can understand its purpose.
+
+![A Code Editor embedded in a form following a set of radio buttons. It has the heading "Policy code (Sentinel)" immediately before.](/assets/components/code-editor/code-editor-external-do-accessible-name.png)
+
+!!!
+
+!!! Dont
+
+Assume all users will understand the purpose of the Code Editor without providing an accessible name. (Not providing one may cause confusion and is an accessibility failure.)
+![A Code Editor embedded in a form following a set of radio buttons. The Code Editor has no title labelling it.](/assets/components/code-editor/code-editor-dont-external-accessible-name.png)
+
+!!!
+
+## Secondary actions
+
+The secondary actions section supports two optional buttons: [Copy Button](/components/copy/button) and a button to expand the editor to fill the viewport. The Copy Button copies the content of the Code Editor to the clipboard, while the second button maximizes and minimizes the Code Editor from inline to full screen. These actions are considered supporting functions less vital to the user’s work but may be useful.
+
+![The secondary actions container is shown twice, both with the Copy Button and full screen button. First showing the full screen button with the maximize icon and the second showing it with the minimize icon.](/assets/components/code-editor/code-editor-secondary-actions.png)
+
+## Custom yielded actions
+
+This space is reserved for primary custom yielded actions. Primary actions are defined as necessary for the user to complete their work.
+
+![The Code Editor with the title "CodeEditor title". The custom yielded element section shows a placeholder and is between the title and the editor.](/assets/components/code-editor/code-editor-yielded-actions-placeholder.png)
+
+!!! Do
+
+![The Code Editor with the title "CodeEditor title". The custom yielded section has a "Reveal secrets" toggle.](/assets/components/code-editor/code-editor-primary-yielded-actions.png)
+
```

Comment:
This "Do" section is missing text. (It's difficult to understand the intention with only an image as the content.)